src/include/platform/ConfigurationManager.h

```diff
@@ -60,6 +61,12 @@ class ConfigurationManager
         kMaxPairingCodeLength      = 16,
         kMaxSerialNumberLength     = 32,
         kMaxFirmwareRevisionLength = 32,
+#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
+        kMACAddressLength = 8,
+#else
+        kMACAddressLength = 6,
```

Comment:
Nit: We're not declaring that all MAC addresses on these platforms are 8 bytes (Thread) and 6 bytes (non-Thread) respectively.  We're declaring that we choose an 802.15.4 phy on Thread platforms to be the primary MAC and we choose an 802.11 phy on non-Thread platforms to be the primary MAC, and we use the primary MAC as an identity.

The following naming more clearly reflects that:

```suggestion
#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
        kPrimaryMACAddressLength = 8,
#else
        kPrimaryMACAddressLength = 6,
```